app/bundles/CoreBundle/Form/Type/AbstractFormStandardType.php

```diff
@@ -111,8 +111,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             );
 
             if (!$builder->has('publishUp') && method_exists($options['data'], 'getPublishUp')) {
-                $builder->add('publishUp', PublishUpDateType::class, ['attr' => ['readonly' => $readonly]]);
-                $builder->add('publishDown', PublishDownDateType::class, ['attr' => ['readonly' => $readonly]]);
+                $builder_options = [
```

Comment:
The preferred style for variable names is camelCase. So, based on that, $builderOptions would be more in line with those guidelines.